prime-router/docs/release-notes.md

```diff
@@ -2,8 +2,9 @@
 
 ## General useful links:
 
-- All Schemas are documented here:  [Link to detailed schema dictionaries](./schema_documentation)
+- All Schemas are documented here: [Link to detailed schema dictionaries](./schema_documentation)
 - The ReportStream API is documented here: [Hub OpenApi Spec](./openapi.yml)
+- More detailed notes about individual releases: [Recent releases](https://github.com/CDCgov/prime-reportstream/releases)
```

Comment:
Instead of "more detailed notes" call it change log like you are in the files.  Try to keep these consistent.

Reply:
absolutely, thank you!